Gruntfile.js

```diff
@@ -144,8 +146,12 @@ module.exports = function(grunt) {
 
     grunt.config.set("gh-release.release-notes", releaseNotes);
 
-    grunt.file.write(dest, "" + header + contents);
-    grunt.log.ok("Changelog updated, and release notes extracted.");
+    if (dryRun) {
+      grunt.log.ok('changelog (dry): Prepending header to ' + dest);
+    } else {
+      grunt.file.write(dest, "" + header + contents);
+    }
+    grunt.log.ok('changelog: ' + dest + ' updated, and release notes extracted.');
```

Comment:
Not really because it's really a notification that the task has completed. We can certainly reword it though.